nrf52840.resc

```diff
@@ -0,0 +1,18 @@
+:name: NRF52840
+:description: This script runs Zephyr Shell demo on NRF52840.
+
+using sysbus
+
+$name?="nRF52840"
+mach create $name
+
+machine LoadPlatformDescription @platforms/cpus/nrf52840.repl
```

Comment:
please put the repl file in this repo as well, so that it's self-contained

Reply:
and adjust the path ofc

Reply:
Done.